pkl-doc/src/main/kotlin/org/pkl/doc/DocPackageInfo.kt

```diff
@@ -165,7 +165,7 @@ data class DocPackageInfo(
             .split("/")
             .map { it.uriEncoded }
             .joinToString("/") { it } + ".pkl"
-        URI(importUri).resolve(path)
+        if (importUri.startsWith("package")) URI(importUri + path) else URI(importUri).resolve(path)
```

Comment:
We can actually just use `URI(importUri + path)` in all cases.

Looks like I was the one that broke this logic in [an earlier PR](https://github.com/apple/pkl/commit/dddae1190febb148821a8560ffafbbba7dba413a#diff-08d9462f05ae0598bb1e90c634ccf7295edb7119c9bfc29d4a83fe93c9bd0ca9R162-R169)

```suggestion
        URI(importUri + path)
```